xtask/codegen/js.ungram

```diff
@@ -2125,14 +2125,43 @@ TsMappedTypeReadonlyModifierClause =
 	operator_token: ('+' | '-')?
 	'readonly'
 
+// a: typeof import("mod", { with: { "resolution-mode": "import" } })
+//                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+TsImportTypeAssertion = 
+  assertion_kind: ('assert' | 'with')
+  ':'
+  '{'
+  assertions: JsImportAssertionEntryList
+  '}'
+
+// a: typeof import("mod", { with: { "resolution-mode": "import" } })
+//                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+TsImportTypeAssertionBlock = 
+	'{'
+	type_assertion: TsImportTypeAssertion
+	'}'
+
+// a: typeof import("mod", { with: { "resolution-mode": "import" } })
+//                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+TsImportTypeArguments = 
+  '('  
+     argument: AnyTsType
+	  (
+		','
+		TsImportTypeAssertionBlock
+	  )?
+  ')'
```

Comment:
There's mixed indentation. Please stick to spaces, and please follow the same style we have in the rest of the file

Reply:
ok，i update here~